### PR TITLE
Beta Raidfilter

### PR DESCRIPTION
**Pull request by jepke**

### CODE REVIEW

File: lib/RocketMap_MAD.php [hunk 1]
```diff
@@ -623,6 +655,12 @@ public function generated_exclude_list($type)
             foreach ($pokestops as $pokestop) {
                 $data[] = $pokestop['incident_grunt_type'];
             }
+	} elseif ($type === 'raidbosslist') {
+            $gyms = $db->query("SELECT distinct pokemon_id FROM raid WHERE pokemon_id > 0 AND end > UTC_TIMESTAMP() order by pokemon_id;")->fetchAll(\PDO::FETCH_ASSOC);
+            $data = array();
+            foreach ($gyms as $gym) {
+                $data[] = $gym['raid_pokemon_id'];
```
Comment (clburlison) [inline lib/RocketMap_MAD.php]: Change this to
```php
$data[] = $gym['pokemon_id']
```

After that everything else is working on my instance

Comment (jepke) [reply lib/RocketMap_MAD.php]: check

**Comment by clburlison** [comment]
Not sure if it is planned but an option to completely remove the "Minimum Raid Level", "Maximum Raid Level", and "Only Active Raids" would be nice. Directing users to use the new raid filter seems like the right thing to do, and removes some extra clutter IMO. Right now we only have the option to set their defaults unless I'm reading incorrectly?

**Comment by jepke** [comment]
> Not sure if it is planned but an option to completely remove the "Minimum Raid Level", "Maximum Raid Level", and "Only Active Raids" would be nice. Directing users to use the new raid filter seems like the right thing to do, and removes some extra clutter IMO. Right now we only have the option to set their defaults unless I'm reading incorrectly?

I believe all those have config variables to turn them off active raids certainly does have that.

**Comment by whitewillem** [comment]
No. Min/max level and only active cant be disabled atm. We should prolly add some.

**Comment by clburlison** [comment]
Posting feedback on the latest commits so they don't get lost in discord. Pulled latest commits. The broken egg filtering is working. Not super useful imo but I guess it gives me a way to check of my raid workers are falling behind. Normally don't have those eggs for more than 60-90 seconds.

**Comment by jepke** [comment]
I think most maps won't have hatched eggs longer than a couple minutes. But leaving them out the filtering will also generate odd situations by completely hiding them until the are scanned for their bosses.

**Pull request merged**